lib/ansible/utils/display.py

```diff
@@ -207,11 +211,11 @@ def locking_wrapper(*args, **kwargs):
     buffer = tio.buffer
 
     # monkeypatching the underlying file-like object isn't great, but likely safer than subclassing
```

Comment:
I wonder what was the problem with subclassing...

Reply:
@mkrizek do we know what this comment implies? I'd rather disagree with the statement made.

Reply:
That would be a question for @nitzmahone and not something related to this PR :-)